src/core/layout/qgslayoutmanager.cpp

```diff
@@ -330,6 +275,25 @@ bool QgsLayoutManager::accept( QgsStyleEntityVisitorInterface *visitor ) const
   return true;
 }
 
+void QgsLayoutManager::setupObjectConnections( QgsMasterLayoutInterface *layout )
+{
+  // ugly, but unavoidable for interfaces...
```

Comment:
reviewer note: this is existing ugly code, moved from QgsLayoutManager::addLayout